iris/meta.yaml

```diff
@@ -11,7 +11,7 @@ build:
 
 requirements:
     build:
-        - python >=2.7,<3
+        - python
```

Comment:
That is already in the run requirements, but it may be we need cf_units at build time for iris (as it imports itself to build the netcdf pyke rules, so build-requirements >= run-requirements)